scripts/portfolioView.js

```diff
@@ -0,0 +1,13 @@
+var portfolioView = {};
+
+$('#projectsContainer').hide();
+
+portfolioView.handelhandleMainNav = function () {
```

Comment:
Great job getting your nav bar to work with showing and hiding respective sections! 👍